Packs/Endace/Integrations/Endace/Endace.py

```diff
@@ -0,0 +1,1336 @@
+    Function Params:
+    app: Endace App Client
+    args: string , archive filename
+    Returns:
+        Dictionary context data in response to the command execution
+    Raises:
+        ValueError: If input argument is in wrong format.
+    """
+
+    if len(args.values()):
+        function_arg = dict()
+        #   archive file name
+        function_arg['archived_filename'] = args.get("archived_filename")
+
+        if re.fullmatch(r'[\w0-9_-]+', args.get("archived_filename")) is None:
+            raise ValueError("Wrong value of archive_filename")
+
+        #   calling archive file delete task function of app instance
+        result = app.delete_archived_file(function_arg)
+
+        #   create entry context to return to Demisto
+        output = {'Endace.ArchivedFile.Delete(val.FileName == obj.FileName)': result}
+        table_header = ['Task', 'FileName', 'Status', 'Error']
+        readable_output = tableToMarkdown('EndaceResult', result, headers=table_header, removeNull=False)
+        raw_response = result
+        return readable_output, output, raw_response
+    else:
+        raise ValueError("Incorrect arguments provided")
+
+
+#  Download Command
+def endace_download_pcap_command(app, args):
+    """ download pcap file function
+       Function Params:
+        app: Endace App Client
+        args: dict of
+               filename: Name of the file to download
+               filesize: limit download to max filesize.
+       Returns: Dictionary context data in response to the command execution
+       Raises:
+           ValueError: If input argument is in wrong format.
+       """
+    if len(args.values()):
+        function_args = {"filename": args.get("filename"), "filesizelimit": args.get("filesizelimit")}
+
+        #   Doing a sanity check on input function arguments
+        try:
+            int(function_args['filesizelimit'])
+        except ValueError:
+            raise ValueError("Filesize Limit value is incorrect")
+        else:
+            if int(function_args['filesizelimit']) < 0:
+                raise ValueError("Filesize Limit value is incorrect")
+
+            #   calling download PCAP function of app instance
+            result = app.download_pcap(function_args)
+
+            #   create entry context to return to Demisto
+
+            output = {'Endace.Download.PCAP(val.FileName == obj.FileName)': result}
+            table_header = ['Task', 'FileName', 'Status', 'Error', 'FileSize', 'FileType', 'FileUser', 'FileURL']
+            readable_output = tableToMarkdown('EndaceResult', [result], headers=table_header, removeNull=False)
+            raw_response = result
+            return readable_output, output, raw_response
+    else:
+        raise ValueError("Incorrect arguments provided")
+
+
+#  Test Command
+def endace_test_command(appurl, username, password, insecure):
+    if re.match(r'https://[\w\-\.]+\.\w+', appurl) is None:
+        raise ValueError('Wrong Appliance URL')
```

Comment:
please provide the correct appliance url format

Reply:
@Itay4 - are you asking to amend the ValueError ? During setting up of an instance, clicking on the "?" icon shows its definition. Is that not indicative enough?

Reply:
no, as it is not shown to the user what might be wrong with the url
the error message should be something like `Wrong Appliance URL. Make sure the URL is in the format ...`